### DISCUSSION TITLE
Welcome to AddisMapTransit official support and discussions

### DISCUSSION THREAD

**Discussion by amenk** [AddisMap/AddisMapTransit#4] [Announcements]
## 👋 Welcome to AddisMapTransit app Support
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Receive support for the Android and iOS app, directly from the team or from the community.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

[Click here to open a new support question.](https://github.com/AddisMap/AddisMapTransit/discussions/new?category=support)

In case you do not want to create a GitHub account, you can also email info@addismap.com for support.

See also [AddisMapTransit.com](https://AddisMapTransit.com/) for more information.

**Comment by abdujemal** [comment]
Really really. Thank you guys!! This is great idea and great implementation. You guys are amazing. I always struggle to go some where. As a developer i always want to build something like this because i struggled alot. But please you guys have to work on the marketing side. I was disappointed to see that you have a couple of downloads. Other than that Really wish you best of luck.  Thank you very much.

**Reply by amenk** [reply]
Thanks, feel free to tell your friends about the project or post on social media.

**Reply by abdujemal** [reply]
Sure i will. How are you guys monitizing this service?

**Reply by amenk** [reply]
It's not monetized, but we had received a grant (DT4A) in the past.
